docs/development/Building in Windows 10 with Linux Subsystem.md

```diff
@@ -2,17 +2,53 @@
 
 Linux subsystem for Windows 10 is probably the simplest way of building INAV under Windows 10.
 
-1. Enable WSL (_Windows Subsystem for Linux) using any guide from internet. [This](https://winaero.com/blog/enable-wsl-windows-10-fall-creators-update/) is up to date step by step (January 2018)
-1. From _Windows Store_ install `Ubuntu`
-1. Start `Ubuntu` and run:
-1. `sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa`
-1. `sudo apt-get update`
-1. `sudo apt-get install gcc-arm-embedded make ruby`
+## Setting up the environment
 
-At the moment (January 2018) it will install `gcc-arm-none-eabi` in version _7 q4_
+Enable WSL:
+run `windows features`
+enable `windows subsytem for linux`
+reboot
 
-From this moment INAV can be build using the following command
 
-`make TARGET={TARGET_NAME}`
+Install Ubuntu:
+1.  Go to Microsoft store https://www.microsoft.com/en-gb/store/b/home
+1.  Search and install most recent Ubunto LTS version
+1.  When download completed, select `Launch Ubuntu`
+1.  When prompted enter a user name and password which you will need to remember
+1.  When complete, the linux command prompt will be displayed
 
-Of course, replace `{TARGET_NAME}` with a target you want to compile
\ No newline at end of file
+NOTE: from this point all commands are entered into the Ubunto shell command window
+
+Update the repo packages:
+1.  `sudo apt update`
+
+Install Git, Make, gcc and Ruby
+1.  `sudo apt-get install git`
+1.  `sudo apt-get install make`
+1.  `sudo apt-get install gcc-arm-none-eabi`
+1.  `sudo apt-get install ruby`
+
+## Downloading the iNav repository (example):
+
+Mount MS windows C drive and clone iNav
+1.   `cd /mnt/c`
+1.   `git clone https://github.com/iNavFlight/inav.git`
```

Comment:
Why do this in Windows, where you haven't been instructed to install git, rather than in Ubuntu, where you have?

Reply:
I was making a bit of an assumption here that this would be for windows users who wanted an easy environment they can edit etc in more familiar windows editors.

Linux users probably already know what they are doing?  :)